### PR TITLE
More `v0.26.0` patches

### PR DESCRIPTION
**Pull request by tinker-michaelj**
**Description**:
 - 🛑 Disables contract expiration and postpones free 90-day renewals to a subsequent release.
    * Also does not publish auto-renew migration records.
 - Per HIP-415, uses 32-byte _prefix_ of SHA-384 record running hash for EVM block hash (not suffix).
 - Uses `0.26.0` SDK.
 - Allows `0.0.<evm_address>` form of `ContractID` to be used with `getAccountBalance` query.
 - Isolates all expiration checks in `ContextOptionValidator`, fixing several cases where enabling contract expiry would have prevented expired accounts from being used.
 - Increase `contracts.maxGas` as a stop-gap, pending its final removal.
 - Standardizes the names of the `MapValueListUtils` helpers.

### CODE REVIEW

File: hedera-node/src/main/java/com/hedera/services/ServicesState.java [hunk 1]
```diff
@@ -167,9 +167,10 @@ public void migrate() {
 					KvPairIterationMigrator::new,
 					VirtualMapMigration::extractVirtualMapData,
 					vmFactory.apply(JasperDbBuilder::new).newVirtualizedIterableStorage());
-			// Grant all contracts one free ~90 day auto-renewal upon enabling contract expiry
-			autoRenewalMigrator.grantFreeAutoRenew(this, getTimeOfLastHandledTxn());
 			ownedNftsLinkMigrator.buildAccountNftsOwnedLinkedList(accounts(), uniqueTokens());
+
+			// When enabling expiry, we will grant all contracts a ~90 day auto-renewal via the autoRenewalMigrator
```
Comment (Neeharika-Sompalli) [inline hedera-node/src/main/java/com/hedera/services/ServicesState.java:172]: Are we not enabling expiry this release ?

File: hedera-node/src/main/resources/bootstrap.properties [hunk 2]
```diff
@@ -53,14 +53,14 @@ contracts.allowCreate2=true
 contracts.defaultLifetime=7890000
 contracts.knownBlockHash=
 contracts.localCall.estRetBytes=32
-contracts.maxGas=5000000
+contracts.maxGas=8000000
```
Comment (Neeharika-Sompalli) [inline hedera-node/src/main/resources/bootstrap.properties:56]: Isn't this 4M ? I thought so from this [issue](https://github.com/hashgraph/hedera-fee-tool-js/issues/101)

**Review by Neeharika-Sompalli — APPROVED** [review]
LGTM other than few questions ! 💯

File: hedera-node/src/main/java/com/hedera/services/ServicesState.java (hunk 1 above)
Comment (Neeharika-Sompalli) [reply hedera-node/src/main/java/com/hedera/services/ServicesState.java:172]: Nvmd .. saw the PR description ! 😞

File: hedera-node/src/main/resources/bootstrap.properties (hunk 2 above)
Comment (tinker-michaelj) [reply hedera-node/src/main/resources/bootstrap.properties:56]: We are (almost certainly) going to remove this entirely, but large factory contracts (e.g. used by SaucerSwap) need 8M gas for certain transactions that do a lot of calls to system contracts.

File: hedera-node/src/main/java/com/hedera/services/ServicesState.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (tinker-michaelj) [reply hedera-node/src/main/java/com/hedera/services/ServicesState.java:174]: Done!

**Pull request merged**